python/paddle/v2/framework/tests/test_evaluator.py

```diff
@@ -4,6 +4,7 @@
 import unittest
 import op_test
 import numpy as np
+exit(0)
```

Comment:
Should be removed?

Reply:
No, it's a newly merged code and contains a bug. I am refactoring it.